assets/css/screen.css

```diff
@@ -749,14 +733,14 @@ The knock-on effect of this is ugly browser-scroll bars at the bottom, so 80px o
 .search-result-title {
   font-size: 14px;
   line-height: 22px;
-  color: #121A24;
+  color: #121a24;
 }
 
 .search-result-meta {
   font-weight: normal;
   font-size: 12px;
   line-height: 16px;
-  color: #7B848F;
+  color: #7b848f;
```

Comment:
seeing these hex values updated throughout makes me wonder if there are color vars we could use for them instead (or if they're just one-off usages in the styling)

Reply:
I could check but.. I have not😇